.github/workflows/build-test-vm.yaml

```diff
@@ -85,7 +86,7 @@ jobs:
 
       - name: build ${{ needs.tags.outputs.vm-postgres-16-bullseye }}
         run: |
-          ./bin/vm-builder -src postgres:16-bullseye -spec tests/e2e/image-spec.yaml -dst ${{ needs.tags.outputs.vm-postgres-16-bullseye }} -daemon-image ${{ needs.tags.outputs.daemon }}
+          ./bin/vm-builder -src postgres:16-bullseye -spec tests/e2e/image-spec.yaml -dst ${{ needs.tags.outputs.vm-postgres-16-bullseye }} -daemon-image ${{ needs.tags.outputs.daemon }} -target-arch linux/${TARGET_ARCH}
```

Comment:
Out of curiosity, why do we need `TARGET_ARCH` / `-target-arch` if

> Has no cross-build support, produced image architecture is the same as host machine architecture.

(sorry if I missed this in the comments/discusstions)